src/latex.jl

```diff
@@ -56,36 +66,59 @@ render(demo; pathname = "latex.gif")
 ```
 
 """
-function latex(text::LaTeXString, pos::Point, draw_object::Symbol)
+function latex(
+    text::LaTeXString,
+    pos::Point,
+    draw_object::Symbol;
+    valign = :top,
+    halign = :left,
+)
     object = CURRENT_OBJECT[1]
     opts = object.opts
     t = get(opts, :draw_text_t, 1.0)
-    return animate_latex(text, pos, t, draw_object)
+    return animate_latex(text, pos, t, valign, halign, draw_object)
 end
 
-function animate_latex(text, pos::Point, t, object)
+function animate_latex(text, pos::Point, t, valign::Symbol, halign::Symbol, object)
     svg = get_latex_svg(text)
     object == :stroke && (object = :fill)
+
+    w, h = svgwh(svg)
+    halignment = findfirst(isequal(halign), (:left, :center, :right, :centre))
+
+    if halignment === nothing
```

Comment:
We would like to cover every single line using our test cases.
You can test for warnings using `@test_logs`